IDEAS/Buildings/Components/Examples/RectangularZone.mo

```diff
@@ -0,0 +1,39 @@
+within IDEAS.Buildings.Components.Examples;
+model RectangularZone
+  "Example that compares a zone with internal wall and without internal wall"
+  import IDEAS;
+  extends Modelica.Icons.Example;
+  Validation.Cases.Case900Template zoneIntWal(
+    hasIntZone=true,
+    lIntZone=zoneIntWal.lA*3,
+    redeclare IDEAS.Examples.PPD12.Data.InteriorWall18 conTypIntZone)
+                                              "Zone with internal wall"
+    annotation (Placement(transformation(extent={{-24,-60},{20,-18}})));
+  Validation.Cases.Case900Template zone "Unshaded zone"
+    annotation (Placement(transformation(extent={{-22,18},{20,60}})));
+  inner BoundaryConditions.SimInfoManager       sim
```

Comment:
by convention we put the `SimInfoManager` at the top of the file.

Reply:
done